### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by kingthorin**
Fixes zaproxy/zaproxy-website#2708

Note omerlh.info also recently started failing, I've reached out to Omer via Twitter DM.

### CODE REVIEW

**Review by psiinon — APPROVED** [review]

**Comment by kingthorin** [comment]
@ricekot / @thc202 nudge 😄

**Comment by kingthorin** [comment]
Of course now that I've hit you up for reviews, Omer got back to me to say that he's abandon his blog. 

Do we want to just remove this or shall I find it via WayBackMachine?
https://github.com/zaproxy/zaproxy-website/blob/b424b2f4c72a5398987a0f1e48c219554477fe0c/site/content/faq/can-zap-be-used-to-test-mobile-apps.md?plain=1#L22

**Comment by thc202** [comment]
If it's accessible, +1.

**Comment by kingthorin** [comment]
The one I quoted above isn't the one that was coming up in the action. I should have listed the one from the evangelists page. In which case using the wayback URL probably doesn't make sense.

**Comment by kingthorin** [comment]
Adjsuted

File: site/content/faq/can-zap-be-used-to-test-mobile-apps.md [hunk 1]
```diff
@@ -19,4 +19,4 @@ And these articles:
 
   * [Intercepting Android traffic using OWASP ZAP](https://thezero.org/blog/2016/01/25/android_proxy_zap/) \- TheZero blog
   * [Four Ways to Bypass Android SSL Verification and Certificate Pinning](https://blog.netspi.com/four-ways-bypass-android-ssl-verification-certificate-pinning/) \- NetSPI Blog
-  * [Debugging iOS apps with Zaproxy](https://www.omerlh.info/2019/05/15/debugging-ios-apps-with-zaproxy/) \- Omer Levi Hevroni's blog
+  * [Debugging iOS apps with Zaproxy](http://web.archive.org/web/20240418164405/https://www.omerlh.info/2019/05/15/debugging-ios-apps-with-zaproxy/) \- Omer Levi Hevroni's blog - Via The Way Back Machine
```
Comment (ricekot) [inline site/content/faq/can-zap-be-used-to-test-mobile-apps.md:22]:
```suggestion
  * [Debugging iOS apps with Zaproxy](https://web.archive.org/web/20240418164405/https://www.omerlh.info/2019/05/15/debugging-ios-apps-with-zaproxy/) \- Omer Levi Hevroni's blog - Via The Way Back Machine
```

**Comment by kingthorin** [comment]
Accepted

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**